back-end/API/app/controllers/enterprise_controller.py

```
@@ -38,41 +38,46 @@ def getEnterpriseByEmployer(id):
             else:
-                logger.warn("An user tried to modify an entreprise don't have permission")
+                logger.warning("An user tried to modify an entreprise don't have permission")
                 return jsonify({'message': 'User cannot modify enterprise'}), 401
-
-        
-        logger.warn(f'Enterprise not found with id : {id}')
-        return jsonify({'message': 'enterprise not found'}), 404
+            
     except Exception as e:
-        logger.error(f'There was an error updating the enterprise')
+        logger.exception(f'There was an error updating the enterprise: {str(e)}')
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)